CHANGELOG.md

```diff
@@ -275,7 +279,7 @@ Additions to existing modules
   uv≈wx⇒yu∙v≈yw∙x       : ∀ y → (y ∙ u) ∙ v ≈ (y ∙ w) ∙ x
   uv≈wx⇒u∙vy≈w∙xy       : ∀ y → u ∙ (v ∙ y) ≈ w ∙ (x ∙ y)
   uv≈wx⇒yu∙vz≈yw∙xz     : ∀ y z → (y ∙ u) ∙ (v ∙ z) ≈ (y ∙ w) ∙ (x ∙ z)
-```
+  ```
```

Comment:
Did you really mean to add these spaces here? I doubt it.

Reply:
I think that markdown rendering automatically indents inside a list.

Reply:
Oh, OK (I've maybe learnt something), but still: consistency with every other entry?

Reply:
Consistency (very) frequently wins such arguments...